sys/include/od.h

```diff
@@ -34,7 +34,7 @@ extern "C" {
 
 /**
  * @brief Dumps memory stored at *data* byte-wise up to *data_len* in
- *        hexadecimal representation to stdout with
+ *        hexadecimal and ASCII representation to stdout
```

Comment:
Oh and maybe add a hint about the `od_string` module here or better in the detailed description of the function.